### PR TITLE
Warn admins to harden Keycloak before publishing the platform

### PR DESCRIPTION
**Pull request by delvh**

### CODE REVIEW

File: docs/install-manuals/publishing-keycloak.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Making keycloak production ready
+
+By default, Misarch creates a few test accounts inside the `Misarch` realm with easy logins.\
+In particular, those are
+
+- `admin`
+- `seller`
```
Comment (nk-coding) [inline docs/install-manuals/publishing-keycloak.md:7]: do we really need this default user?

Comment (nk-coding) [reply docs/install-manuals/publishing-keycloak.md:7]: same for test
in particular for the k8 deployment, I would prefer if these are not there

**Review by nk-coding — APPROVED** [review]

File: docs/install-manuals/publishing-keycloak.md (hunk 1 above)
Comment (delvh) [reply docs/install-manuals/publishing-keycloak.md:7]: It is meant for convenience:
If you are a researcher who simply wants to try something (our largest user base), you're happy if you do not need to create the user every time yourself…
That's why I think we need those default users.

**Review by lesser1 — APPROVED** [review]
I don't think that I have the expertise to judge the correctness or quality of the page but overall it looks good to me.

File: docs/install-manuals/publishing-keycloak.md (hunk 1 above)
Comment (legendofa) [reply docs/install-manuals/publishing-keycloak.md:7]: What is the test user able to do, is it a "customer" or "shop-user" user?

File: docs/install-manuals/publishing-keycloak.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- `test`
+
+You should delete these accounts before publishing the platform to keep the permission system intact.
+
+Furthermore, you should change the password of the **Keycloak** `admin` account in the `master` realm so that no external person can access the Keycloak console.\
```
Comment (legendofa) [inline docs/install-manuals/publishing-keycloak.md:12]: Is there a way to require a password reset, to make it secure by default?

Comment (delvh) [reply docs/install-manuals/publishing-keycloak.md:12]: Yes, but I've explicitly disabled it

**Review by legendofa — APPROVED** [review]

File: docs/install-manuals/publishing-keycloak.md (hunk 1 above)
Comment (delvh) [reply docs/install-manuals/publishing-keycloak.md:7]: A normal user who can only buy things

**Pull request merged**